### ISSUE TITLE
Mangled records returned from bolt driver.

### ISSUE THREAD

**Issue by holmok** [neo4j/neo4j-javascript-driver#64]
local neo4j version: 3.0.0-M05
driver version 1.0.0

code:

```
'use strict';
var neo4j = require('neo4j-driver').v1;
var driver = neo4j.driver("bolt://localhost", neo4j.auth.basic("neo4j", "-----"));

var session = driver.session();

session.run("MATCH (n:ATTRACTION) RETURN n LIMIT 20;")
    .subscribe(
        {
            onNext: function (record) {
                var node = record.get('n');
                console.log(node);
            },
            onCompleted: function () {
                // Completed!
                session.close();
            },
            onError: function (error) {
                console.log(error);
            }
        }
    );
```

all the nodes look fine, but one looks like this:

```
Node {
  identity: Integer { low: 4373, high: 0 },
  labels: 
   [ '�q��N�\u0011\u0015��',
     Integer { low: 65, high: 0 },
     Integer { low: 84, high: 0 },
     Integer { low: 84, high: 0 } ],
  properties: Integer { low: 82, high: 0 } }
```

it's a fairly random smattering.  any ideas?

**Comment by pontusmelke** [comment]
Hi @holmok,
We have never encountered this before, is it always the same node (`identity: 4373`) or does it happen to different nodes? If it only happens to that node could you add `WHERE id(n) = 4373` and also try to do `console.log(n.labels[0])`?

Thanks for reporting this!

**Comment by vmarovic** [comment]
Hi,
I seem to have same issue:

Record {
  keys: [ 'u._id' ],
  length: 1,
  _fields: [ '�q��$d5ceb4ea-e6f9-4397-823b-103cf4a' ],
  _fieldLookup: { 'u._id': 0 }
}

It does seem to happen on the same nodes always, and only when returning large number of records. When i fetch this record only it is returned properly. Also if i skip to location near this record and limit results to 10 it is returned ok also.

**Comment by pontusmelke** [comment]
Ok thanks, we'll look into this issue

**Comment by vmarovic** [comment]
Hi just one more thing
If i write query like 
match (u:User) return u.name
I get some names mangled, but if i typecast it to string in cypher
match (u:User) return toString(u.name)
results are returned fine.

**Comment by pontusmelke** [comment]
That doesn't look right. With _some names_, do you mean that is it the same as in the other example that it only happens when there is a large number of records?

**Comment by vmarovic** [comment]
Yes it only seems to happen when large number of records is returned. Also errors seem to be evenly spaced out... in the first example with uuid mangled records are on positions 182, 364, 564 in the results array.

**Comment by pontusmelke** [comment]
We have released version 1.0.1 which should adress this issue. Thanks for reporting and thanks @kanban-agencey for helping out.
